src/main/java/org/folio/consortia/config/AppConfig.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.consortia.config;
+
+import org.folio.consortia.domain.converter.TenantConverter;
+import org.folio.consortia.domain.converter.UserTenantConverter;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.format.FormatterRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class AppConfig implements WebMvcConfigurer {
+
+  @Override
+  public void addFormatters(FormatterRegistry registry) {
```

Comment:
Actually I think we can use simpler way of converters declaration, but lets improve it in next PRs to block subsequent endpoints

Reply:
Ok